ibm_catalog.json

```diff
@@ -153,19 +184,71 @@
                 }
               ]
             },
-            {
-              "key": "service_credential_secrets"
-            },
             {
               "key": "skip_secrets_manager_cos_iam_auth_policy"
             },
             {
               "key": "cos_instance_cbr_rules"
+            },
+            {
+              "key": "provider_visibility",
+              "options": [
+                {
+                  "displayname": "private",
+                  "value": "private"
+                },
+                {
+                  "displayname": "public",
+                  "value": "public"
+                },
+                {
+                  "displayname": "public-and-private",
+                  "value": "public-and-private"
+                }
+              ]
             }
-          ]
+          ],
+          "dependencies": [
+            {
+              "name": "deploy-arch-ibm-account-infra-base",
+              "description": "Cloud automation for Account Configuration organizes your IBM Cloud account with a ready-made set of resource groups by default—and, when you enable the \"with Account Settings\" option, it also applies baseline security and governance settings.",
+              "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+              "flavors": [
+                "resource-group-only",
+                "resource-groups-with-account-settings"
+              ],
+              "default_flavor": "resource-group-only",
+              "id": "63641cec-6093-4b4f-b7b0-98d2f4185cd6-global",
+              "input_mapping": [
+                {
+                  "dependency_output": "workload_resource_group_name",
+                  "version_input": "existing_resource_group_name"
+                },
+                {
+                  "dependency_input": "prefix",
+                  "version_input": "prefix",
+                  "reference_version": true
+                },
+                {
+                  "dependency_input": "provider_visibility",
+                  "version_input": "provider_visibility",
+                  "reference_version": true
+                },
+                {
+                  "version_input": "use_existing_resource_group",
```

Comment:
I am pretty sure COS DA was updated not to use this boolean anymore

Reply:
Copied from KMS DA -> https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/blob/1c67113828c0700556f6d2420a6f6e9217920bca/ibm_catalog.json#L299-L300

So a bug in that config too - Ill create PR to fix that